http-client/ingredients-database_utils.ts

```diff
@@ -0,0 +1,72 @@
+import {renderProduct} from '../products_utils'
+import {ingredientData} from '../type/ingredient-data'
+
+export const getAllIngredients = async (ingredientsArray: Array<ingredientData>) => {
+    const loader = $('<div>').addClass('products__loader').appendTo($('.products'))
+    const url = 'http://localhost:3001/ingredients'
```

Comment:
you can create a util function to append only the last part to the url. Localhost is mandatory so, you shouldn't repeat it multiple times